test/relations.test.js

```diff
@@ -605,6 +605,864 @@ describe('relations', function() {
       db.automigrate(['Physician', 'Patient', 'Appointment', 'Address'], done);
     });
 
+    describe('scoped queries', function() {
+      describe('find', function() {
+        it('should allow to use fields on related model', function(done) {
+          Physician.create(function(err, physician) {
```

Comment:
Unhandled `err`.

Please use async/await style in new tests, juggler supports only Node.js 8+ version now.

```ts
const physician = await Physician.create();
```

Reply:
(The same comment applies to all other places where you are invoking a function or a method and providing a callback.)

Reply:
Fixed